src/vs/workbench/contrib/chat/common/chatService.ts

```diff
@@ -316,6 +316,19 @@ export interface IChatTerminalToolInvocationData {
 	terminalToolSessionId?: string;
 	/** The predefined command ID that will be used for this terminal command */
 	terminalCommandId?: string;
+	/** Serialized URI for the command that was executed in the terminal */
+	terminalCommandUri?: UriComponents;
+	/** Serialized output of the executed command */
+	terminalCommandOutput?: {
+		text: string;
+		truncated?: boolean;
+		isHtml?: boolean;
+	};
```

Comment:
I always envisioned this being done in xterm.js instead of HTML when we do streaming, we need to do something like that to avoid the problems we're having now in chat with slow markdown rendering.

So we may end up changing this

Reply:
Yes, makes sense that we will change this when it's xterm vs html